source/configure/calls-deployment.rst

```diff
@@ -356,3 +366,65 @@ Can the traffic between Mattermost and ``rtcd``  be kept internal or should it b
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When possible, it's recommended to keep communication between the Mattermost cluster and the dedicated ``rtcd`` service under the same private network as this can greatly simplify deployment and security. There's no requirement to expose ``rtcd``'s HTTP API to the public internet.
+
+Debugging
```

Comment:
Wondering why this was moved here. The rationale to having it in the `make-calls` page was the this kind of information would be especially useful to end users struggling to connect. Having it here implies that its target audience is the admin configuring/deploying calls which is not always the case in my opinion. Happy to hear more thoughts though.

Reply:
Fair point! I thought it was more something an admin would do but if it's for the end users then I can do some content review to make it more friendly.